packages/serverpod_serialization/test/websocket_messages_test.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:serverpod_serialization/src/websocket_messages.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test(
+      'Given a Ping command message when building websocket message from string then PingCommand is returned.',
+      () {
+    var message = PingCommand.buildMessage();
+    var result = WebSocketMessage.fromJsonString(message);
+    expect(result, isA<PingCommand>());
+  });
+
+  test(
+      'Given a Pong command message when building websocket message from string then PongCommand is returned.',
+      () {
+    var message = PongCommand.buildMessage();
+    var result = WebSocketMessage.fromJsonString(message);
+    expect(result, isA<PongCommand>());
+  });
+
+  test(
+      'Given an unknown command json String when building websocket message from string then UnknownMessage is returned.',
+      () {
+    var message = '{"messageType": "this is not a known message type"}';
+    var result = WebSocketMessage.fromJsonString(message);
+    expect(result, isA<UnknownMessage>());
+  });
+
+  test(
+      'Given an invalid json String when building websocket message from string then UnknownMessage is returned.',
+      () {
+    var message = 'This is not a valid json string';
+    var result = WebSocketMessage.fromJsonString(message);
+    expect(result, isA<UnknownMessage>());
+  });
+
+  test(
+      'Given a null messageType when building websocket message from string then UnknownMessage is returned.',
+      () {
+    var message = '{"messageType": null}';
+    var result = WebSocketMessage.fromJsonString(message);
+    expect(result, isA<UnknownMessage>());
+  });
```

Comment:
Suggestion: Add a test with an unknown type as well. E.g. 

```
var message = '{"messageType": "unknown_type"}';
```

Maybe test/deal with case insensitivity? "PONG_COMMAND" should that be valid or not?